modules/proc_manage-ldap-setup.adoc

```diff
@@ -30,45 +30,112 @@ created. Restart the {productname} container and log in to the Superuser
 panel *_as the user that was just created._*
 
 [[setup-ldap-configuration]]
-== Setup LDAP Configuration
-
-Navigate to the Superuser panel and navigate to settings section. Locate
-the Authentication section and select “LDAP” from the drop-down menu.
-
-image:authentication-ldap.png[Select LDAP from the Authentication section]
-
-Enter LDAP configuration fields as required.
-
-image:authentication-ldap-details.png[Fill in LDAP information]
-
-[[few-tips-for-ldap-configuration]]
-== Tips for LDAP configuration:
-
-* LDAP URI must be in ldap:// or ldaps:// syntax. Typing a URI with
-ldaps:// prefix will surface the option to provide custom SSL
-certificate for TLS setup
-* User Relative DN is relative to BaseDN (ex: ou=NYC not
-ou=NYC,dc=example,dc=org)
-* Logged in Username must exist in User Relative DN
-* You can enter multiple “Secondary User Relative DNs” if there are
-multiple Organizational Units where User objects are located at. (ex:
-ou=Users,ou=NYC and ou=Users,ou=SFO). Simply type in the Organizational
-Units and click on Add button to add multiple RDNs
-* sAMAccountName is the UID attribute for against Microsoft Active
-Directory setups
-* {productname} searches "User Relative DN" with subtree scope. For
-example, if your Organization has Organizational Units NYC and SFO under
-the Users OU (`ou=SFO,ou=Users and ou=NYC,ou=Users`), {productname} 
-can authenticate users from both the NYC and SFO Organizational Units if
-the User Relative DN is set to Users (ou=Users)
+== Set Up LDAP Configuration
+
+In the config tool, locate the Authentication section and select “LDAP” from the drop-down menu. Update LDAP configuration fields as required.
+
+image:authentication-ldap.png[Fill in LDAP information]
+
+* Here is an example of the resulting entry in the _config.yaml_ file:
+....
+AUTHENTICATION_TYPE: LDAP
+....
+
+=== Full LDAP URI
+
+image:authentication-ldap-uri.png[LDAP server URI]
+image:authentication-ldap-ssl.png[LDAP server SSL]
+
+* The full LDAP URI, including the _ldap://_ or _ldaps://_ prefix.
+* A URI beginning with _ldaps://_ will make use of the provided SSL certificate(s) for TLS setup.
```

Comment:
there're two locations on Quay-config app to upload customized certificate, need to let customers know which one is valid for LDAPS.